diff --git a/lsusb.py b/lsusb.py
index 14d9a31..a34f736 100755
--- a/lsusb.py
+++ b/lsusb.py
@@ -16,7 +16,7 @@ nohub = False
 warnsort = False
 
 prefix = "/sys/bus/usb/devices/"
-usbids = "/usr/share/usb.ids"
+usbids = "/lib/hwdata/usb.ids"
 
 esc = chr(27)
 norm = esc + "[0;0m"
diff -up usbutils-004/Makefile.am.orig usbutils-004/Makefile.am
--- usbutils-004/Makefile.am.orig	2011-09-20 09:42:04.282566870 +0200
+++ usbutils-004/Makefile.am	2011-09-20 09:42:41.399102845 +0200
@@ -38,7 +38,7 @@ EXTRA_DIST = \
 	usbutils.pc.in
 
 lsusb.8: $(srcdir)/lsusb.8.in
-	sed 's|VERSION|$(VERSION)|g;s|@usbids@|$(datadir)/usb.ids|g' $< >$@
+	sed 's|VERSION|$(VERSION)|g;s|@usbids@|/lib/hwdata/usb.ids|g' $< >$@
 
 usb-devices.1: $(srcdir)/usb-devices.1.in
 	sed 's|VERSION|$(VERSION)|g' $< >$@
@@ -47,7 +47,7 @@ pkgconfigdir = $(datarootdir)/pkgconfig
 pkgconfig_DATA = usbutils.pc
 
 usbutils.pc: $(srcdir)/usbutils.pc.in
-	sed 's|@usbids@|$(datadir)/usb.ids|g;s|@VERSION[@]|$(VERSION)|g' $< >$@
+	sed 's|@usbids@|/lib/hwdata/usb.ids|g;s|@VERSION[@]|$(VERSION)|g' $< >$@
 
 DISTCLEANFILES = \
 	lsusb.8 \
--- usbutils-006/configure.ac~	2012-07-06 23:20:41.000000000 +0300
+++ usbutils-006/configure.ac	2013-01-20 00:04:33.521843802 +0200
@@ -41,7 +41,7 @@
 	datadir:                ${datadir}
 	mandir:                 ${mandir}
 
-	usb.ids:                ${datadir}/usb.ids
+	usb.ids:                /lib/hwdata/usb.ids
 	zlib:                   ${USE_ZLIB}
 
 	compiler:               ${CC}
