--- usbutils-018/lsusb.py.orig	2024-10-22 11:33:33.000000000 +0200
+++ usbutils-018/lsusb.py	2024-10-23 13:18:12.108103599 +0200
@@ -27,9 +27,7 @@
 
 prefix = "/sys/bus/usb/devices/"
 usbids = [
-	"/usr/share/hwdata/usb.ids",
-	"/usr/share/misc/usb.ids",
-	"/usr/share/usb.ids",
+	"/lib/hwdata/usb.ids",
 ]
 cols = ("", "", "", "", "", "")
 
@@ -467,7 +466,7 @@
 	print("  -C, --no-color        disable colors")
 	print("  -e, --endpoints       display endpoint info")
 	print("  -f FILE, --usbids-path FILE")
-	print("                        override filename for /usr/share/{hwdata/,}usb.ids")
+	print("                        override filename for " + ", ".join(usbids))
 	print("  -w, --wakeup          display power wakeup setting")
 	print()
 	print("Use lsusb.py -ciu to get a nice overview of your USB devices.")
