--- usbutils-010/lsusb.py.in.orig	2018-05-20 16:28:37.757075652 +0200
+++ usbutils-010/lsusb.py.in	2018-05-20 16:30:24.147074439 +0200
@@ -519,7 +519,7 @@
 	print(" -c use colors")
 	print(" -e display endpoint info")
 	print(" -w display warning if usb.ids is not sorted correctly")
-	print(" -f FILE override filename for /usr/share/usb.ids")
+	print(" -f FILE override filename for " + usbids)
 	print("Use lsusb.py -ciu to get a nice overview of your USB devices.")
 	return 2
 
--- usbutils-010/Makefile.am.orig	2018-05-15 15:45:26.000000000 +0200
+++ usbutils-010/Makefile.am	2018-05-20 15:33:58.907113099 +0200
@@ -51,11 +51,11 @@
 	usbreset.c
 
 lsusb.py: $(srcdir)/lsusb.py.in
-	sed 's|VERSION|$(VERSION)|g;s|@usbids@|$(datadir)/usb.ids|g' $< >$@
+	sed 's|VERSION|$(VERSION)|g;s|@usbids@|/lib/hwdata/usb.ids|g' $< >$@
 	chmod 755 $@
 
 lsusb.8: $(srcdir)/lsusb.8.in
-	sed 's|VERSION|$(VERSION)|g;s|@usbids@|$(datadir)/usb.ids|g' $< >$@
+	sed 's|VERSION|$(VERSION)|g;s|@usbids@|/lib/hwdata/usb.ids|g' $< >$@
 
 usb-devices.1: $(srcdir)/usb-devices.1.in
 	sed 's|VERSION|$(VERSION)|g' $< >$@
--- usbutils-010/configure.ac.orig	2018-05-20 11:34:32.067277134 +0200
+++ usbutils-010/configure.ac	2018-05-20 15:35:00.030445728 +0200
@@ -35,7 +35,7 @@
 	datadir:                ${datadir}
 	mandir:                 ${mandir}
 
-	usb.ids:                ${datadir}/usb.ids
+	usb.ids:                /lib/hwdata/usb.ids
 
 	compiler:               ${CC}
 	cflags:                 ${CFLAGS}
